src/freenet/l10n/freenet.l10n.de.properties

```diff
@@ -1420,14 +1449,20 @@ PproxyToadlet.pluginGroup.example=Beispiele
 PproxyToadlet.pluginGroup.file-transfer=Datei-Transfer
 PproxyToadlet.pluginGroup.index=Indexieren
 PproxyToadlet.pluginGroup.technical=Technisches
+PproxyToadlet.pluginGroupTitle=${pluginGroup} Plugins
 PproxyToadlet.pluginVersion=Version
 PproxyToadlet.pluginNotFoundReload=Das angegebene Plugin konnte nicht gefunden werden, um es neu zu laden.
 PproxyToadlet.pluginNotFoundReloadTitle=Plugin nicht gefunden (neu laden)
 PproxyToadlet.pluginNotFoundUpdatingTitle=Neuste Version des Plugins nicht gefunden!
 PproxyToadlet.pluginNotFoundUpdating=Die neuete Version für das Plugin wurde nicht gefunden und konnte deshalb nicht geladen werden!
+PproxyToadlet.pluginSourceChoice=Sie können die Plugins entweder über Freenet oder über unsere zentralen Server laden.
+PproxyToadlet.pluginSourceFreenet=Abruf über Freenet. Dies ist schwer zu verfolgen, sicher und kann langsam sein.
+PproxyToadlet.pluginSourceHTTPS=Lade über das Web von unserem zentralen Server.
+PproxyToadlet.pluginSourceHTTPSWarning=WARNUNG: Dies wird zeigen, dass Sie Freenet nützen, und wird nicht funktionieren, wenn unsere Website geblockt ist! Es könnte für einen Abhörer sogar möglich sein, zu erraten, was Sie laden wollen, und es mit z.B. dem Erstellen eine Freemail Identität zu verbinden.
```

Comment:
This one (and probably some of the ones right above this) also doesn’t exist anymore.

Reply:
fixed - thank you!